openshift/topics/introduction.adoc

```diff
@@ -2,6 +2,7 @@
 
 === What Is {project_name}?
 {project_name} is an integrated sign-on solution available as a Red Hat JBoss Middleware for OpenShift containerized image. The {project_openshift_product_name} image provides an authentication server for users to centrally log in, log out, register, and manage user accounts for web applications, mobile applications, and RESTful web services.
+{project_name} is only available on the following platforms: *{openshift_image_platforms}*.
```

Comment:
Suggestion, maybe add cross-reference across the two docs? 

E.g. something aligned with the following: `{project_name} is only available on the following platforms: *{openshift_image_platforms}*. See link_to:{project_name_counterpart_platform_docs_guide_url} documentation for further information about the {project_name_counterpart_platform_image_name}.`

And for both of the guides define `{project_name_counterpart_platform}` as the complement of `["x86_64", "s390x"] - {openshift_image_platforms}`. Similarly `{project_name_counterpart_platform_image_name}`.

WDYT?

Reply:
Good idea. Addressed in the updated version.